modules/xz/5.4.5/MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "xz",
+    version = "5.4.5",
+    compatibility_level = 1,
```

Comment:
@meteorcloudy What is the best practice here? Should we use `compatibility_level = 5,` since the major version here is "5"? Or should we start with  `compatibility_level = 0,`? `compatibility_level = 5,` would make the most sense for me, since this leaves the door open if someone wants to use 4.x, 3.x, 2.x, 1.x for some reason in the future.

Reply:
I would recommend starting with the default value, i.e. don't specify this attribute. Whether you want to bump the compatibility for a new major version depends on how breaking the changes are. If the build targets stay the same, it's probably better not to bump. In any case, this can be deicded later.

Reply:
Will do!